shared/src/sys/out.rs

```diff
@@ -62,3 +62,26 @@ pub mod crypto {
         pub fault: u32,
     }
 }
+
+pub mod vm {
+    use crate::clock::ChainEpoch;
+    use crate::sys::TokenAmount;
+    use crate::{ActorID, MethodNum};
+
+    #[derive(Debug, Copy, Clone)]
+    #[repr(packed, C)]
+    pub struct InvocationContext {
+        /// The value that was received.
+        pub value_received: TokenAmount,
+        /// The caller's actor ID.
+        pub caller: ActorID,
+        /// The receiver's actor ID (i.e. ourselves).
+        pub receiver: ActorID,
+        /// The method number from the message.
+        pub method_number: MethodNum,
+        /// The current epoch.
+        pub network_curr_epoch: ChainEpoch,
```

Comment:
Can drop the `network_` prefix here as there's no namespacing needed any longer, nor is there ambiguity.

Reply:
@Stebalien wanted the prefix back, so the two of you have to fight that one out

Reply:
Ah, sorry, I wanted the `network_` prefix for the _version_ (because we have many versions). The _epoch_ is unambiguous.